pkgs/by-name/re/realvnc-vnc-viewer/darwin.nix

```
@@ -12,7 +12,7 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   src = fetchurl rec {
       name = "VNC-Viewer-${finalAttrs.version}-MacOSX-universal.dmg";
       url = "https://downloads.realvnc.com/download/file/viewer.files/${name}";
-      sha256 = "0k72fdnx1zmyi9z5n3lazc7s70gcddxq0s73akp0al0y9hzq9prh";
+      sha256 = "1i9mly54qqxhiy6z9p0q8px3n1rc014vdxjzsmn3mx25q11gd845";
```

```suggestion
      hash = "";
```

Use SRI hash instead (you have to regenerate it)

---

regenerate via `nix hash to-sri --type sha256`